### PR TITLE
Issue 819

### PR DESCRIPTION
**Pull request by durgeshanaokar**
Change in pkg/controller/zerocapacity/zero_controller.go.zeroPodSpec() method add:
// Add Custom labels missing from zero capacity pods
	labels := zeroSpec.PodLabels
	ispn.AddLabelsForPods(labels)

### CODE REVIEW

**Comment by dmvolod** [comment]
@durgeshanaokar please rebase and open on top of the latest master

File: pkg/controller/zerocapacity/zero_controller.go [hunk 1]
```diff
@@ -328,11 +328,14 @@ func (z *Controller) zeroPodSpec(name, namespace, configMap string, podSecurityC
 		return nil, err
 	}
 	dataVolName := name + "-data"
+	// Add Custom labels missing from zero capacity pods
```
Comment (ryanemerson) [inline pkg/controller/zerocapacity/zero_controller.go:331]: No need for the comment as it won't be missing with your changes :slightly_smiling_face:

File: pkg/controller/zerocapacity/zero_controller.go [hunk 2]
```diff
@@ -405,6 +408,7 @@ func (z *Controller) zeroPodSpec(name, namespace, configMap string, podSecurityC
 
 	ispnCtrl.AddVolumeForUserAuthentication(ispn, &pod.Spec)
 	ispnCtrl.AddVolumeForEncryption(ispn, &pod.Spec)
+
```
Comment (ryanemerson) [inline pkg/controller/zerocapacity/zero_controller.go:411]: Unnecessary whitespace change, can be removed.

**Comment by durgeshanaokar** [comment]
Closing the PR as some issues with checking will raise another one.
